resources/setups/coupler/base_config.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
What's this for?  The coupler should be a drop-in functional replacement for other switch implementations... why a separate setup config?

Reply:
Just something I am using to test for now. Will get rid of it.